assets/stylesheets/components/_modules.scss

```diff
@@ -26,3 +26,4 @@
 @import 'video';
 @import 'crds-components-skeleton-blocks';
 @import 'logo';
+@import 'header-box';
```

Comment:
Instead of creating a new file for this, can you please add it to the following section within `_typography.scss`?

https://github.com/crdschurch/crds-styles/blob/feature/US19600-header-box/assets/stylesheets/components/_typography.scss#L126